source/community.applications/usr/local/emhttp/plugins/community.applications/include/force_update_helpers.php

```
@@ -0,0 +1,111 @@
+        . "</span></font><font size='3'><br><br>Community Applications requires your server to have internet access.  This could be because it appears that the current date and time of your server is incorrect.  Correct this within Settings - Date And Time.  See also <a href='https://forums.unraid.net/topic/120220-fix-common-problems-more-information/page/2/?tab=comments#comment-1101084' target='_blank'>this post</a> for more information";
+    }
+
+    $tempFile = @file_get_contents(CA_PATHS['appFeedDownloadError']);
+    $downloaded = @file_get_contents($tempFile);
+
+    if (strlen($downloaded) > 100) {
+      $response['data'] .= "<font size='2' color='red'><br><br>It *appears* that a partial download of the application feed happened (or is malformed), therefore it is probable that the application feed is temporarily down.  Please try again later)</font>";
+    }
+
+    $response['data'] .= "<div class='ca_center'>Last JSON error Recorded: ";
+    $jsonDecode = json_decode($downloaded, true);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$jsonDecode'. (undefined)

(UnusedLocalVariable)